ethers-core/src/types/trace/geth.rs

```diff
@@ -0,0 +1,47 @@
+use crate::types::H256;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct GethTrace {
+    failed: bool,
+    gas: i64,
+    #[serde(rename = "returnValue")]
+    return_value: String,
```

Comment:
This is an array of bytes [geth ExecutionResult](https://github.com/ethereum/go-ethereum/blob/3fd16af5a91e232f1b32c85db79c45d5563ae14f/core/state_transition.go#L84) that gets marshalled into a string [geth Logger](https://github.com/ethereum/go-ethereum/blob/d8a2305565b1f97c451f8595e0f65358d6842714/eth/tracers/logger/logger.go#L247) before becoming JSON. So, I think you're right -- bytes[] makes sense. (Even though this could be a revert string, seems like getting the bytes[] is more natural than getting the string representation of the bytes[] return value)

Reply:
[aa42e1c](https://github.com/gakonst/ethers-rs/pull/1469/commits/aa42e1cb6ccd3597b3cd3b15b2e0cfcf186969ea)